MainDemo.Wpf/ColorZones.xaml

```diff
@@ -4,13 +4,16 @@
              xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
              xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
              xmlns:materialDesign="http://materialdesigninxaml.net/winfx/xaml/themes"
+             xmlns:codeDisplayer="clr-namespace:CodeDisplayer;assembly=CodeDisplayer"
              mc:Ignorable="d" 
              d:DesignHeight="800" d:DesignWidth="600">
     <ScrollViewer>
-        <StackPanel Margin="16">
-            <TextBlock FontSize="16">Colour Zones</TextBlock>
-            <TextBlock TextWrapping="Wrap" Margin="0 16 0 0">The ColorZone control allows you to  easily define striking blocks of colour to give your application extra clarity and style, whilst still remaining within the bounds of your Material Design palette.</TextBlock>
-            <TextBlock Margin="0 16 0 0" FontSize="10">Invert the basic paper/body colours.</TextBlock>
+        <codeDisplayer:XamlDisplayerPanel >
```

Comment:
Firstly, where did this .dll/control come from?  I don't really want to be checking in .dll files.  Who owns it, can we get it up on nuget?

Reply:
@ButchersBoy I wrote the library. Ok, let me upload in to nuget.

Reply:
Cool...are you going to open source it on GitHub?  I would appreciate knowing the source is around, and pretty sure other projects would benefit from this.

Reply:
Yes, it is open source already. You can see the code [here](https://github.com/wongjiahau/Displaying-XAML)

Reply:
Awesome!

OK, if you get it set up as a proper nuget package we'll get this pulled in.  :+1:

Reply:
Sorry...didn't read your previous comment about nuget!

Reply:
This is the nuget link to XamlDisplayer https://www.nuget.org/packages/XamlDisplayerPackage/ .
But  I'm not sure what I'm still lacking yet as this is my first published nuget package.